collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/ConfigureWithQRCodeTest.java

```diff
@@ -113,23 +113,6 @@ public void clickConfigureQRCode_opensScanner_andThenScanning_importsSettings()
                 .assertText("http://gallops.example");
     }
 
-    @Test
-    public void clickConfigureQRCode_opensScanner_andThenScanning_whenUsingOldQR_importsSettingsAndMigrates() {
```

Comment:
This is covered by the regression tests put in place when we found a bunch of broken edge cases. It's really hard to test here and get much value (other than we get from the normal import test).